src/ts/contentScripts/dispatcher/index.ts

```diff
@@ -20,6 +20,20 @@ const dispatchMap = new Map([
 
 browser.runtime.onMessage.addListener((command) => dispatchMap.get(command)?.());
 
+document.addEventListener('keydown', ev => {
+    const enter = 'Enter';
```

Comment:
not sure if this constant adds much value tbh

Reply:
Yeah. I initially tried using the event.keyCode, which is much less clear in the condition check, but then neglected to inline the constant again when I switched to `ev.key`.